test/T60.const.hpp

```diff
@@ -0,0 +1,48 @@
+// -*- mode:c++;tab-width:2;indent-tabs-mode:t;show-trailing-whitespace:t;rm-trailing-spaces:t -*-
+// vi: set ts=2 noet:
+//
+// Copyright (c) 2016 Sergey Lyskov <sergey.lyskov@jhu.edu>
+//
+// All rights reserved. Use of this source code is governed by a
+// MIT license that can be found in the LICENSE file.
+
+/// @file   binder/test/T00.basic.hpp
+/// @brief  Binder self-test file. Basic functionality.
+/// @author Sergey Lyskov
+
+#ifndef _INCLUDED_T60_basic_hpp_
+#define _INCLUDED_T60_basic_hpp_
+#include <string>
+#include <vector>
+
+const int global_int=1;
+const long global_long=2;
+
+const unsigned int global_unsigned_int=3;
+const unsigned long global_unsigned_long=4;
+
+const float global_float=5.0;
+const double global_double=6.0;
+
+const std::string global_string1="Some test string";
+
+const std::string global_string2=std::string("Some test string");
+
+
+const double expression_global_double=8.0+1.0/5.0;
```

Comment:
Yes, that is a problematic part.

> we can somehow evaluate them

Eh, it is safer not to evaluate them. Just imagine a code
```
#ifdef MYFLAG
int const a=5;
#else
int const a=6;
#endif
```
and then somewhere binding  the constants like
```
int const i_want_to_bind_it   mya = 100 * a;
```
In case the evaluation would be turned on, one would have to regenerate the binding code each time the 'a' is updated or the MYFLAG is changed.

What could make more sense is to transform the RHS from the "raw" code into a canonical representation. I will have to look how to do that (LLVM should be able to do that, or am I wrong?).